p2p/server.py

```diff
@@ -246,11 +142,11 @@ def _make_peer_pool(self, discovery: DiscoveryProtocol) -> PeerPool:
 
     async def _run(self) -> None:
         self.logger.info("Running server...")
-        upnp_dev = await self._discover_upnp_device()
-        external_ip = '0.0.0.0'
-        if upnp_dev is not None:
-            external_ip = upnp_dev.WANIPConn1.GetExternalIPAddress()['NewExternalIPAddress']
-            await self._add_nat_portmap(upnp_dev)
+        mapped_external_ip = await self.upnp_service.add_nat_portmap()
+        if mapped_external_ip is None:
+            external_ip = '0.0.0.0'
```

Comment:
Wondering if we can move away from this and instead do what geth does and just not show an IP address.